presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/Util.java

```diff
@@ -14,8 +14,11 @@
 package com.facebook.presto.sql.planner.iterative.rule;
```

Comment:
But in case you copy commit message, then this commit would be eligible to be merged from this PR . Then, once it got merged, you would not need copy-paste this commit to other PRs.